R/functions.R

```diff
@@ -1,4 +1,4 @@
-# source("R/rhelpers.R")
+source("R/rhelpers.R")
```

Comment:
For Pete's sake, NO.

*Never* *ever* use source in a package.

Reply:
Yes, I see
Won't use it again